scripts/version.py

```diff
@@ -32,6 +32,22 @@ def explore_plugin_version_name():
 
 
 def explore_plugin_version():
+    # kolibri-explore-plugin v6.27.0 and newer store the version in a separate
+    # file, so try that first. Then fall back to parsing __init__.py, which is
+    # what older versions used.
+    try:
+        with open(
+            f"./{EXPLOREPLUGIN_TARGET}/kolibri_explore_plugin/_version.py", "r"
+        ) as version_file:
+            # The _version.py file always has the plugin version between quotes,
+            # in an assignment to __version__:
+            lines = version_file.readlines()
+            for line in lines:
+                if line.startswith("__version__ = "):
+                    return line.split("'")[1]
+    except FileNotFoundError:
+        pass
```

Comment:
Just a drive-by comment: given this file is a Python module that defines a constant, and given this program reading it is a Python module, we could import it:

https://docs.python.org/3/library/importlib.html#importing-a-source-file-directly

Reply:
Done